spec/factories/models/pages.rb

```diff
@@ -10,7 +10,7 @@ def as_json(*args)
 
 FactoryBot.define do
   factory :page, class: "Page" do
-    association :form
+    id { Faker::Number.number(digits: 2) }
```

Comment:
does this risk overlapping with other pages generated within the same test?

Reply:
Yes, but I'm not sure it matters at the moment - we aren't using it anywhere else, and in this PR it's only used to stop `goto_page_options` returning `id: nil` for every page in the form. But the tests will still pass if we remove this line so I don't mind removing it if we think the potential name clash is a concern.

Reply:
I think its fine for the moment, we will be visiting this again when we add more functionality